shell/platform/android/io/flutter/view/FlutterView.java

```diff
@@ -72,7 +74,7 @@
  * <p>Deprecation: {@link io.flutter.embedding.android.FlutterView} is the new API that now replaces
  * this class. See https://flutter.dev/go/android-project-migration for more migration details.
  */
-public class FlutterView extends SurfaceView implements BinaryMessenger, TextureRegistry {
+public class FlutterView extends SurfaceView implements BinaryMessenger, TextureRegistry, MouseCursorPlugin.MouseCursorViewDelegate {
```

Comment:
All changes made in the V1 android embedding should also be mirrored in the V2 embedding found at android/io/flutter/embedding/android/FlutterView.java

Reply:
I think I did? https://github.com/flutter/engine/pull/18569#discussion_r429098654